CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 1.0.1-dev
```

Comment:
Flip this to `1.1.0-dev` – adding new features!

Reply:
will do!

Reply:
done.